packages/runtime/container-runtime/src/gc/index.ts

```diff
@@ -16,6 +16,7 @@ export {
 	gcVersionUpgradeToV2Key,
 	IGarbageCollectionRuntime,
 	IGarbageCollector,
+	IGarbageCollectorConfigs,
```

Comment:
Why is this exported?

Reply:
To be used in tests. It's not exported by container-runtime package.

Reply:
woah... `gc/index.ts`.  Missed that sub-path.  Why is this index.ts needed?   (we can discuss offline)